docker/girder.Dockerfile

```diff
@@ -1,24 +1,40 @@
-FROM girder/girder:latest
+FROM node:latest as builder
+WORKDIR /app
+# Install dependencies
+COPY client/package.json client/yarn.lock /app/
+RUN yarn --frozen-lockfile
+# Build
+COPY .git/ /app/.git/
+COPY client/ /app/
+RUN yarn build
```

Comment:
Works fine as intended when checking out this specific branch.  We may want to add something to the instructions that makes an actual `git clone` a requirement or part of the process to build/run this.  I know it isn't common but if you download a zip and extract it you will definitely get an error during the build process because of the lack of a .git folder.

Reply:
Noted in README.md